### PR TITLE
Grant permissions to Zoom

### PR DESCRIPTION
**Pull request by zoomdevplatform**
# Description

https://github.com/jenkinsci/zoom-plugin.git
https://issues.jenkins-ci.org/browse/HOSTING-726
# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

File: permissions/plugin-zoom.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "zoom"
+github: "jenkinsci/zoom-plugin"
+paths:
+- "org/jenkins-ci/plugins/zoom"
+developers:
+- "zoomdevplatform"
```
Comment (batmat) [inline permissions/plugin-zoom.yml]: https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=zoomdevplatform is 404

Comment (zoomdevplatform) [reply permissions/plugin-zoom.yml]: @batmat Fixed. Thanks!

Comment (zoomdevplatform) [reply permissions/plugin-zoom.yml]: Hi @batmat, can you help to try again?

**Comment by zoomdevplatform** [comment]
Hi @batmat @jenkinsadmin, is there any other errors?

File: permissions/plugin-zoom.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: "zoom"
+github: "jenkinsci/zoom-plugin"
+paths:
+- "org/jenkins-ci/plugins/zoom"
```
Comment (batmat) [inline permissions/plugin-zoom.yml]: Wrong path. It should match the plugin's groupId. I.e. `io.jenkins.plugins`

Comment (zoomdevplatform) [reply permissions/plugin-zoom.yml]: @batmat Sorry, is that all right now?

**Comment by zoomdevplatform** [comment]
Hi @batmat, do you know why i still have no access?
image

**Comment by batmat** [comment]
Seems like an unrelated issue. Bad SCM url format maybe? On the phone right now, hard to check

**Pull request merged**